### PR TITLE
chore: downstream static analysis fixes for ts

### PR DESCRIPTION
**Pull request by danielmoy-google**
Caveat: I don't know typescript at all, I'm just trusting the linters.

### CODE REVIEW

File: kythe/typescript/indexer.ts [hunk 1]
```diff
@@ -230,7 +230,7 @@ class SymbolVNameStore {
   get(symbol: ts.Symbol, ns: TSNamespace, context: Context): VName|undefined {
     if (this.store.has(symbol)) {
       const nsCtx = this.serialize(ns, context);
-      return this.store.get(symbol)!.get(nsCtx);
+      return this.store.get(symbol).get(nsCtx);
```
Comment (evmar) [inline kythe/typescript/indexer.ts]: I am surprised by this change.  What tool thinks this is right?

Comment (danielmoy-google) [reply kythe/typescript/indexer.ts]: sending you a pointer to the downstream analyzer result

Comment (danielmoy-google) [reply kythe/typescript/indexer.ts]: abandoning this in favor of #4003 then?

Comment (evmar) [reply kythe/typescript/indexer.ts]: Yeah I will fix this in g3.

**Pull request closed without merging**